index.html

```diff
@@ -5,11 +5,161 @@
+                </ul>
+            </nav>
+        </div>
+        <section id="home" class="home">
+            <h1>visual communication conference</h1>
+            <p class=date>pasadena, california </p>
+            <p class=date>may 4-6, 2019</p>
+            <div class="logo-container">
+                <div class="circle"></div>
+                <div class="circle"></div>
+            </div>
+            <p class="button"><a href="#" target="_blank">Buy Ticket</a></p>
+        </section>
+
+    </header>
+    <main>
+        <p class="discription">VCC (Visual Communication Conference) is the largest design conference about Visual
+            Communication.
+            We share our opinions, ideas together and try to find out better ways to communicate with people around the
+            world.
+            Experience the design trend through the world and meet with nearly 2,000 attendees from inside and outside
+            of
+            U.S.
+        </p>
+        <!-- https://www.howdesignlive.com/about-->
+        <section id="speakers" class="speakers">
+            <h2>Speakers</h2>
+            <ul class="clearfix">
+
+                <figure class="adam"><img src="images/adam.jpg" alt="Adam Conner-Simons">
+                    <figcaption>
+                        <h3>Adam Conner-Simons</h3>
+                        <p>Head of Communications & Media Relations,Massachusetts Institute of Technology</p>
+                    </figcaption>
+                </figure>
+
+                <figure class="aaron"><img src="images/aaron.jpg" alt="Aaron Ganci">
+                    <figcaption>
+                        <h3>Aaron Ganci</h3>
+                        <p>Associate Professor of Visual Communication Design,
+                            Herron School of Art + Design, IUPUI</p>
+                    </figcaption>
+                </figure>
+
+                <figure class="karen"><img src="images/karen.jpg" alt="Karen Faith">
+                    <figcaption>
+                        <h3>Karen Faith</h3>
+                        <p>Ethnographer, Strategist, Facilitator Karen Faith, Inc
+                        </p>
+                    </figcaption>
+                </figure>
+            </ul>
+        </section>
+
+        <section id="schedule" class="schedule">
+            <table>
+                <tr>
+                    <th>Day1</th>
+                    <th>Topic</th>
+                    <th>Speaker</th>
+                </tr>
+                <tr>
+                    <td>10:00</td>
+                    <td> <a class="button" href="#popup1">How MIT Gets Inventive with Digital Storytellings</a>
+
+                        <div id="popup1" class="overlay">
+                            <div class="popup">
+                                <h2>How MIT Gets Inventive with Digital Storytellings</h2>
+                                <a class="close" href="#">&times;</a>
```

Comment:
I know you pointed out that closing the modal/popup always takes the user up to the top of the site; if you change the `href` here to be your schedule section, you can eliminate that problem:
```suggestion
                                <a class="close" href="#schedule">&times;</a>
```